src/test/java/com/wavefront/opentracing/WavefrontSpanTest.java

```diff
@@ -153,10 +154,10 @@ public void testCustomRedMetricsTagsWavefrontSpan() throws IOException, Interrup
     WavefrontTracer tracer = new WavefrontTracer.Builder(
         new WavefrontSpanReporter.Builder().withSource(DEFAULT_SOURCE).build(wfSender),
         buildApplicationTags()).setReportFrequenceMillis(50).
-        redMetricsCustomTagKeys(new HashSet<>(Arrays.asList("tenant", "env"))).
+        redMetricsCustomTagKeys(new HashSet<>(Arrays.asList("tenant", "env", "kind"))).
```

Comment:
customers won't set it explicitly
span.kind is set implicitly
if it is present as a span tag, then propagate that to RED metrics automatically
without end user stating this explicitly.

Reply:
Thanks for the clarification!